liblangutil/ParserBase.h

```diff
@@ -62,10 +68,17 @@ class ParserBase
 
 	///@{
 	///@name Helper functions
-	/// If current token value is not _value, throw exception otherwise advance token.
+	/// If current token value is not _value, throw exception otherwise advance token
+	//  @a if _advance is true.
```

Comment:
Please also explain that this might not throw an exception if error recovery is activated.